### DISCUSSION TITLE
Difficulty in typing

### DISCUSSION THREAD

**Discussion by Yung-Jer** [legendkong/subi-public#1] [General]
Hi, I would like to point out that the font color is white in color, and the background of the text box is white in color as well. Hence, I have some difficulties of typing in the textbox. Overall, it is really amazing!

**Comment by legendkong** [comment]
Hey Yung-Jer, thanks for pointing it out! I've fixed that issue. This was caused by another fix a few hours ago for screens that are not in dark mode. Cheers!
